CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog].
 
+## Unreleased
+### Enhancements
+* Prefix matching, similar to the completion style `partial`, has been
+  added. As is the case in default Emacs, `t-t-l` matches
+  `toggle-truncate-lines` and `fi--a-po` matches `find-file-at-point`,
+  `find-function-at-point`, and other similarly named symbols. One
+  difference is that you can't use `*` as equivalent to the regexp
+  `.*` (it is instead taken literally), since you can already achieve
+  the same affect by separating your queries with a space.
+
```

Comment:
The changelog entry should, I think, be explicit that this is an optional new filter method that you can use by setting `prescient-filter-method`.